app/assets/javascripts/pieces.js

```diff
@@ -5766,7 +5766,7 @@ pi.utils = (function() {
 
   utils.uniq_id = 100;
 
-  utils.email_rxp = /\b[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}\b/i;
+  utils.email_rxp = /\b[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,15}\b/i;
```

Comment:
I chose fifteen because, it is the longest at the moment (SANDVIKCOROMANT :) )
Open-ended - is good solution to not have problems in the future.